bin/check_tron_jobs.py

```diff
@@ -189,6 +204,12 @@ def check_job_result(job, client, dry_run):
         log.info(result)
     else:
         log.debug("Sending event: {}".format(result))
+        if 'runbook' not in result:
+            log.info("No runbook specified")
+            result['runbook'] = 'foo'
+        if 'team' not in result:
```

Comment:
This needs to be short-circuited early. If we don't have a team we can quickly return.